code-of-conduct.md

```diff
@@ -0,0 +1,88 @@
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation. The following list is
+not meant to be exhaustive, but merely to enumerate some common kinds of
+behaviors.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Reporting & Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the Dask Code of Conduct committee at
+dask-conduct@googlegroups.com or by filling out
+[this form](https://goo.gl/forms/YHc1pc9POZRH7RDt2). Currently, the committee
+consists of:
+
+- James Bourbeau
+- Tom Augspurger
+- Matthew Rocklin
+
+All complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The committee is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+In addition, the online form allows you to submit a report anonymously.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
```

Comment:
Isn't enforcing the CoC the task of the CoC committee, rather than individual maintainers?

Reply:
The CoC committee will handle any instances that are reported through the online form or sent to dask-conduct@googlegroups.com. I think the use of "enforcement" here is referring to the "Project maintainers have the right to remove, edit, or reject comments..." paragraph in the "Our Responsibilities" section.